kitsune/customercare/tasks.py

```diff
@@ -1,11 +1,25 @@
 from celery import task
 from django.contrib.auth.models import User
+
 from kitsune.customercare.zendesk import ZendeskClient
 
 
 @task
-def update_zendesk_user(user_id):
+def update_zendesk_user(user_id: int) -> None:
     user = User.objects.get(pk=user_id)
     if user.profile.zendesk_id:
         zendesk = ZendeskClient()
         zendesk.update_user(user)
+
+
+@task
+def update_zendesk_identity(user_id: int, email: str) -> None:
+    user = User.objects.get(pk=user_id)
+    # email = user.email
+    zendesk_user_id = user.profile.zendesk_id
+    zendesk = ZendeskClient()
```

Comment:
Tests are failing because this is outside the `if user.profile.zendesk_id` conditional